<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1541</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="twMatrix">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>321</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pbRemoveRow">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAddRow">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>280</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAddCol">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>340</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRemoveCol">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbShow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFill">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Заполнить</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="twMatrix_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>371</width>
      <height>321</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pbRemoveRow_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAddRow_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>280</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAddCol_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>340</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRemoveCol_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbShow_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>440</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFill_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>500</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Заполнить</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="twResult">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>10</y>
      <width>411</width>
      <height>331</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pbPLUS">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMINUS">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCOMP">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>220</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbShow_3">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>440</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSIZE">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0x0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSIZE2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>360</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0x0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1541</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
